source/blog/2019-05-11-rspec-rails-4-0-has-been-released.md

```diff
@@ -0,0 +1,171 @@
+
+This means that we no longer support versions of Ruby below 2.2
+in line with the supported Ruby versions of Rails 5.
+
+We have restricted the version of Rails for this version to 4.2 to
+allows a "soft support" for Rails 4.2 during this transitionary
+period but we strongly urge you to upgrade Rails to use 4.0.0.
+
+### Removed version lock with RSpec gems
+
+The `rspec-rails` gem is no longer locked with the versioning of the other RSpec gems, we will now be releasing `rspec-rails` with each new major Rails release, keeping in lockstep with their supported major versions.
+
+### Improved JRuby support
+
+We have improved JRuby support and RSpec Rails 4 fully supports JRuby on Rails 5 and 6.
+
+### Support for Action Cable
+
+We now support [channel specs][1] in order to easily test your Action Cable channels.  A `channel spec` pulls in `ActionCable::Channel::TestCase`, including all of the behavior and assertions that it provides, in addition to RSpec's own behavior and expectations.
+
+### Support for Action Mailbox
+
+Rails 6 came with [Action Mailbox][2]. We can now test them using
+`:mailbox` flag.
+
+```ruby
+```
+
+TODO: We do no have feature test for this...
+
+### Improvements to generators
+
+If you use generators, there are some improvements:
+- The default hash syntax is now the Ruby 1.9 style in generated files to match Rails conventions
+- Request specs are generated instead of controller specs by default
+- New [generators][3] available (channel, generator, mailbox...)
+
+### Allow configuring Active Job `queue_adapter` in system specs
+
+Some people want to run Active Job with a specific queue adapter. In system
+specs you can now choose your desired queue adapter:
+
+```ruby
+before do
+  ActiveJob::Base.queue_adapter = :inline
+end
+```
+
+### Silence log output with Puma
+
+By default, Puma will no longer print logs when running system specs.
+
+### Ability to manually turn off Active Record support
+
+When using Rails without ActiveRecord or using an alternative ORM
+or a database of choice, e.g. Sequel, ROM, Mongoid etc. We can
+mistakenly detect ActiveRecord as being used due to other gems
+autoloading the constants, we now support manually turning off
+Active Record support when not configured with:
+`config.use_active_record = false`
+
+## Stats:
+
+* **Total Commits**: 258
+* **Merged pull requests**: 56
+* **27 contributors**: Andrew White, Anton Rieder, Benoit Tigeot, Jon Rowe,
+David Revelo, Giovanni Kock Bonetti, Ignatius Reza, James Dabbs, Joel AZEMAR,
+John Hawthorn, Jonathan Rochkind, Kieran O'Grady, Moshe Kamensky,
+OKURA Masafumi, Olle Jonsson, Pedro Paiva, Phil Pirozhkov, Penelope Phippen,
+Seb Jacobs, Tanbir Hasan, Viacheslav Bobrov, Vladimir Dementyev, Xavier Shay,
+alpaca-tc, pavel, ta1kt0me
+
+## Docs
+
+### API Docs
+
+* [rspec-rails](/documentation/4.0/rspec-rails/)
+
+### Cucumber Features
+
+* [rspec-rails](https://relishapp.com/rspec/rspec-rails/v/4-0/)
+
+## Release Notes
+
+### 4.0.0 / 2020-03-24
```

Comment:
I feel we should probably include `4.0.1` here now too :joy:

Reply:
Yep! :)